website/docs/docs-section/how-does-it-work/limitations.md

```diff
@@ -3,22 +3,20 @@ title: Limitations
 sidebar_label: Limitations
 ---
 
-In this section, we will discuss the limitations of OpenAttestation credentials.
+This article talks about the limitations of OpenAttestation credentials.
 
-# QR Code
+# QR code
 
-In some cases, it might be beneficial for the data of credentials to be fully contained inside machine-readable content, such as QR Code.
+In some cases, it will be beneficial to add the data of credentials into machine-readable content, such as QR Code.
 
-As we have seen previously, OpenAttestation credentials are naturally verbose. During wrapping, we add a salt in front of each value. Because of this process, the number of data that an OpenAttestation credential can hold to fit inside a QR Code is limited.
+OpenAttestation credentials are naturally verbose. During wrapping, OpenAttestation adds some information in front of each value. Therefore, the number of data that an OpenAttestation credential can hold to fit inside a QR Code is limited.
```

Comment:
Can we keep the clarity of `salt` instead of `some information`?

Also I think "amount of data" is better than "number of data"

Reply:
Yes, will recover "salt". But it looks like we need to add an entry for the definition of "salt" in Glossary.

For the "amount of data", could we change it to "the amount of data fields"? That's because of this GitHub readme: https://github.com/Open-Attestation/open-attestation/tree/master/benchmarks/qr-code

Reply:
In a sense it is both the number of data fields and the length of data that are limited. 

You could have one single field that exceeds the size limit or multiple single character / empty fields that would in total exceed the size limit due to the adding of salt.

Note how in the linked readme we specify the parameter `The values use the provided character set (or Latin by default) and use the provided length (or 20 by default).`

As most of the time a single field will not be so large, the number of data fields is still the more common restriction here.

Reply:
Updated to the following:

> Therefore, there are restrictions on the number of data fields and the length of data that an OpenAttestation credential can hold to fit inside a QR Code.